cadasta/utilities/resources.py

```diff
@@ -95,3 +95,17 @@ def get_ui_class(ui_file):
         )
     )
     return uic.loadUiType(ui_file_path)[0]
+
+
+def get_project_path():
+    """
+    Get absolute project path
+    :return absolute project path
```

Comment:
We also add ':rtype: basetring'
PyCharm will be aware that a string is returned from the function.